docs/Features/heroku.md

```diff
@@ -33,9 +33,9 @@ jhipster heroku
 - Postgres ✔ 
 - MSSQL ✔ (requires a manual step described below)
 
-Currently MySql and PostgreSQL database addons are free and automatically added by this sub-generator when deploying to Heroku. Even though those resouces are free, an accound with verified credit card is required to add them.
+The MySql and PostgreSQL database addons are not free of cost and won't be automatically added by the sub-generator when deploying to Heroku. A paid plan is required to use the resources and addons.
```

Comment:
Some db addons are still free. Other addons (specially those managed by heroku itself) will be paid. The generator will deploy them anyway (except the MSSQL one for now)

So this line can be changed to something like:
`The database addons might not be free of cost and a paid plan is required to use them.`